HerPublicWebsite.BusinessLogic/Models/AnonymisedReport.cs

```diff
@@ -0,0 +1,37 @@
+﻿using HerPublicWebsite.BusinessLogic.Models.Enums;
+
+using HerPublicWebsite.BusinessLogic.Extensions;
+
+namespace HerPublicWebsite.BusinessLogic.Models;
+
+public class AnonymisedReport
+{
+    public string PostcodeFirstHalf { get; set; }
+    public bool IsLsoaProperty { get; set; }
+    public EpcRating EpcRating { get; set; }
+    public DateTime? EpcLodgementDate { get; set; }
+    public bool IsEligible { get; set; }
+    public HasGasBoiler HasGasBoiler { get; set; }
+    public IncomeBand IncomeBand { get; set; }
+    public string LocalAuthorityName { get; set; }
```

Comment:
Sorry, I just realised that this is the DB table rather than the CSV file. Please just put the code in the DB - we can look up the name when writing the CSV file for the DMS to ingest.